### PR TITLE
fix: Use updated wording and slogan

### PR DESCRIPTION
**Pull request by jhermsmeier**
This changes instances of "burn" -> "flash" in order to match Etcher's repository descriptions & slogan

Change-Type: minor
Connects To: https://github.com/resin-io/etcher/issues/797

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const locals = {
   title: 'Etcher',
   prefix: IS_PROD ? '' : '',
   slogan: 'Burn. Better.',
```
Comment (jhermsmeier) [inline config/index.js:22]: Not sure what to do about this one above – I vaguely recall this was discussed at some point, but not sure if we have a new wording for this now? /cc @alisondavis17

File: config/index.js [hunk 2]
```diff
@@ -74,12 +74,12 @@ const locals = {
 		{
 			"title": "More on the way",
 			"image": "features/coming-soon.png",
-			"meta": "50% faster burns, simultaneous writing for multiple drives.<br/><a target='_blank' href='https://github.com/resin-io/etcher/milestones'>View our roadmap</a>"
+			"meta": "50% faster flashs, simultaneous writing for multiple drives.<br/><a target='_blank' href='https://github.com/resin-io/etcher/milestones'>View our roadmap</a>"
```
Comment (lurch) [inline config/index.js]: flashs -> flashes

Comment (jhermsmeier) [reply config/index.js]: Fixed!

File: config/index.js (hunk 1 above)
Comment (alisondavis17) [reply config/index.js:22]: I have nothing off the top of my head but happy to discuss it if needed.

File: config/index.js [hunk 3]
```diff
@@ -74,12 +74,12 @@ const locals = {
...
 		}
 	],
   story: [
-		"Here at <a target='_blank' href='https://resin.io'>resin.io</a> we have thousands of users working through our getting started process and until recently we were embarassed about the steps that involved burning an SD card. There was a separate track for each Mac/Windows/Linux and several manual and error-prone steps along the way.",
-		"To our surprise there was nothing out there that fitted our needs. So we built Etcher, a SD card burner app that is simple for end users, extensible for developers, and works on any platform. "
+		"Here at <a target='_blank' href='https://resin.io'>resin.io</a> we have thousands of users working through our getting started process and until recently we were embarassed about the steps that involved flashing an SD card. There was a separate track for each Mac/Windows/Linux and several manual and error-prone steps along the way.",
+		"To our surprise there was nothing out there that fitted our needs. So we built Etcher, a SD card flasher app that is simple for end users, extensible for developers, and works on any platform. "
```
Comment (Shou) [inline config/index.js]: I know this doesn't specifically pertain to this PR, but there seems to be some inconsistencies with "a SD" and "an SD", with the latter sounding better.

**Review by Shou — APPROVED** [review]

File: config/index.js (hunk 3 above)
Comment (jhermsmeier) [reply config/index.js]: Yeah, true – fixed!

**Pull request merged**